core/controllers/base.py

```diff
@@ -387,7 +390,7 @@ def render_template(
         self.response.pragma = 'no-cache'
 
         self.response.write(self.jinja2_env.get_template(
-            filename).render(**values))
+            'pages/%s' % filename).render(**values))
```

Comment:
Yes, I think so. I'm happy to discuss it, though. Sorry that you may need to revert those lines. D: